app/utils.py

```diff
@@ -29,3 +30,54 @@ def parse_phone_number(phone_number):
         stripped = '1' + stripped
     stripped = '+' + stripped
     return stripped
+
+
+def parse_to_db(db, filename):
+    city_default = ', philadelphia, pennsylvania, usa'
```

Comment:
This is a bit of a hack. The Google geocoding API has [Viewport biasing](https://developers.google.com/maps/documentation/geocoding/intro#Viewports) which would be a more robust way of doing this.